mdsets/core/connector/ExampleConnector.py

```diff
@@ -0,0 +1,18 @@
+from mdsets.core.base.BaseConnector import BaseConnector
+
+
+class ExampleConnector(BaseConnector):
```

Comment:
Examples for abstracts?
![image](https://user-images.githubusercontent.com/23365594/164944057-72178abe-c375-4ec1-8ce8-e9275e7a2c15.png)

Reply:
Feels weird but necessary